dashboard/app/controllers/api/v1/pd/applications_controller.rb

```diff
@@ -3,8 +3,8 @@ class Api::V1::Pd::ApplicationsController < ::ApplicationController
 
   # GET /api/v1/pd/applications
   def index
-    application_data = {}
     regional_partner_id = params[:regional_partner]
+    application_data = {}
 
```

Comment:
This doesn't make that much difference in terms of how close the declaration of `application_data` is to where it gets used, because it has to be defined outside of the `{queries}.each` block below.